L1Trigger/L1CaloTrigger/python/Phase1L1TJetHwEmuComp.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
```

Comment:
this probably belongs in test/ rather than python/

Reply:
moved to test